idaes/gas_solid_contactors/flowsheets/ss_BFB_OC_oxidation.py

```diff
@@ -0,0 +1,171 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2019, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Flowsheet example of the 2-region bubbling fluidized bed (BFB) model
+for a methane combustion with iron-oxide case study
+
+Created: 05/04/2020
+
+Author: Chinedu Okoli
+"""
+# Import python libraries
+import time
+
+# Import Pyomo libraries
+from pyomo.environ import ConcreteModel, SolverFactory, value
+
+# Import IDAES core modules
+from idaes.core import FlowsheetBlock
+
+# Import IDAES logger
+import idaes.logger as idaeslog
+
+# Import BFB unit model
+from idaes.gas_solid_contactors.unit_models.bubbling_fluidized_bed \
+    import BubblingFluidizedBed
+
+# Import property packages
+from idaes.gas_solid_contactors.properties.oxygen_iron_OC_oxidation. \
+    gas_phase_thermo import GasPhaseThermoParameterBlock
+from idaes.gas_solid_contactors.properties.oxygen_iron_OC_oxidation. \
+    solid_phase_thermo import SolidPhaseThermoParameterBlock
+from idaes.gas_solid_contactors.properties.oxygen_iron_OC_oxidation. \
+    hetero_reactions import HeteroReactionParameterBlock
+
+
+# -----------------------------------------------------------------------------
+def main():
+
+    # ---------------------------------------------------------------------
+    # Build model
+
+    # Create a concrete model
+    m = ConcreteModel()
+
+    # Create a steady-state flowsheet
+    m.fs = FlowsheetBlock(default={"dynamic": False})
+
+    # Set up thermo-physical and reaction properties
+    m.fs.gas_properties = GasPhaseThermoParameterBlock()
+    m.fs.solid_properties = SolidPhaseThermoParameterBlock()
+
+    m.fs.hetero_reactions = HeteroReactionParameterBlock(
+            default={"solid_property_package": m.fs.solid_properties,
+                     "gas_property_package": m.fs.gas_properties})
+
+    # Build the BFB in the flowsheet
+    m.fs.BFB = BubblingFluidizedBed(
+            default={
+                    "flow_type": "co_current",
+                    "finite_elements": 5,
+                    "transformation_method": "dae.collocation",
+                    "gas_phase_config":
+                    {"property_package": m.fs.gas_properties},
```

Comment:
It might be because "property_package" is inline with the terminology used in other process simulators. Not sure of the particular reason though as this has been the standard for majority of the IDAES unit models. Maybe @andrewlee94 has a better idea.

Reply:
That is what everything else uses. A better question is why state blocks use "properties".